suite/tests/tools.c

```diff
@@ -297,7 +297,7 @@ print(const char *fmt, ...)
 #    ifdef UNIX
 
 /***************************************************************************/
-/* a hopefuly portable /proc/@self/maps reader */
+/* a hopefully portable /proc/@self/maps reader */
```

Comment:
Update to modern style: capitalized and end-punctuated.

Reply:
Done.